frontend/components/AnimatedLogo.tsx

```diff
@@ -1,396 +1,93 @@
+const getRandomStyles = () =>
+  [
+    `animation-duration: ${Math.ceil(Math.random() * 1400) + 600}ms`,
+    `animation-name: ${
+      [
+        "jsr_logo_square_slide_in_down",
+        "jsr_logo_square_slide_in_left",
+        "jsr_logo_square_slide_in_right",
+        "jsr_logo_square_slide_in_up",
+      ][Math.floor(Math.random() * 4)]
+    }`,
+  ].join(";");
+
+const colors = {
+  "░": "#f7df1e",
+  "█": "#083344",
+} as const;
+
+const squares = `
+  ███████
+  █░█░░░█████
+███░█░███░░░█
+█░█░█░░░█░█░█
+█░░░███░█░███
+█████░░░█░█
+    ███████
+`.split("\n")
+  .filter(Boolean)
+  .map<Array<{ color: keyof typeof colors; x: number; y: number }>>((line, y) =>
+    line.split("").flatMap((color, x) =>
+      color === "█" || color === "░" ? [{ color, x, y }] : []
+    )
+  ).flat();
+
 // Copyright 2024 the JSR authors. All rights reserved. MIT license.
 export function AnimatedLogo() {
   return (
     <svg
       width="100%"
       height="100%"
       id="jsr-logo"
-      viewBox="0 0 833 539"
-      version="1.1"
-      xmlns="http://www.w3.org/2000/svg"
-      xmlnsXlink="http://www.w3.org/1999/xlink"
-      xml:space="preserve"
-      style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;"
+      viewBox="-2 -2 17 11"
```

Comment:
The actual drawing is 13 by 7, but accounting for a padding of 2 all around.

We could write this thusly with less magic numbers:

```js
const padding = 2;
const width = 13;
const height = 7;
const viewBox = [
  -padding,
  -padding,
  padding + width + padding,
  padding + height + padding,
].join(' ');
```